shell/platform/windows/window_binding_handler_delegate.h

```diff
@@ -20,9 +20,12 @@ class WindowBindingHandlerDelegate {
   using KeyEventCallback = std::function<void(bool)>;
 
   // Notifies delegate that backing window size has changed.
-  // Typically called by currently configured WindowBindingHandler, this is
-  // called on the platform thread.
-  virtual void OnWindowSizeChanged(size_t width, size_t height) = 0;
+  //
+  // Called by |FlutterWindow| on the platform thread.
+  //
+  // Returns true if the delegate completed the window resize synchronously.
+  // The return value should only be used for unit testing.
```

Comment:
Not that I can think of a great use for this but any reason we _shouldn't_ use it for anything other than unit tests? If so, consider documenting why that's a bad idea if it can be stated succinctly.

Reply:
I added this comment in case someone wonders why we discard the return value in the implementation. The implementation could use this if needed. I'll tweak the comment to make this clearer.